configure.ac

```diff
@@ -184,6 +184,18 @@ AC_ARG_WITH(
     [with_bundled_protobuf="$withval"],
     [with_bundled_protobuf="detect"]
 )
+AC_ARG_ENABLE(
```

Comment:
this is not correct imho.... the 3rd parameter sets `enable_ml="yes"` even if i give `--disable-ml`

as per AC_ARG_ENABLE docs here https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/Package-Options.html

it adds 2 flags to configure script:
>     --enable-feature[=arg]
>     --disable-feature

Reply:
correct way would be
```
AC_ARG_ENABLE(
    [ml],
    [AS_HELP_STRING([--enable-ml], [Enable anomaly detection @<:@no@:>@])],
    [enable_ml="$enableval"],
    [enable_ml="no"]
)

Reply:
I think I've addressed this with my new changes. Please, reopen if this is not the case.